### PR TITLE
Uprev xsm-policy to support xen 4.18

### PR DESCRIPTION
**Pull request by crogers1**
Linked to https://github.com/OpenXT/xenclient-oe/pull/1488

### CODE REVIEW

File: policy/modules/xen/xen.if [hunk 1]
```diff
@@ -42,7 +54,8 @@ interface(`xen_domain_type',`
 				setvcpucontext getscheduler pause unpause 
 				getvcpuinfo getaddrsize
 				setaffinity getaffinity settime
-				setdomainhandle shutdown set_misc_info trigger};
+				setdomainhandle shutdown set_misc_info trigger
+                getpagingmempool setpagingmempool };
```
Comment (jandryuk) [inline policy/modules/xen/xen.if]: alignment is off.  Looks like you used spaces, but the file is hard tabs.

**Review by jandryuk — CHANGES_REQUESTED** [review]
One minor thing.  Looks good - thanks!

File: policy/modules/xen/xen.if (hunk 1 above)
Comment (crogers1) [reply policy/modules/xen/xen.if]: Gets me every time.  Updated.

**Review by jandryuk — APPROVED** [review]
Thanks!

**Comment by jandryuk** [comment]
Merging tomorrow

**Pull request merged**